### PR TITLE
Add LookAt function to GizmoVisual class

### PR DESCRIPTION
**Pull request by iche033**
# 🎉 New feature

## Summary
The LookAt function updates the rotation visuals in a GizmoVisual so that they always face the specified position. This is used by TransformController so that the rotation visuals (handles) are always positioned in front of the attached object w.r.t the camera so that they are easy to click on by the mouse.

## Checklist
- [x] Signed all commits for DCO
- [x] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: include/gz/rendering/GizmoVisual.hh [hunk 1]
```diff
@@ -69,6 +69,21 @@ namespace gz
...
+      /// \brief Update the Gizmo visual's rotation visuals to face the
+      /// specified position.
+      /// \param[in] _pos Position to face. One typical position to face is
+      /// the eye (camera) position so that the rotation visuals are rendered
+      /// in a pose that is not behind the object the Gizmo visual is
+      /// attached to.
+      /// \param[in] _rot Rotation to be applied. For local space rotation,
+      /// set this to the rotation of the object that the Gizmo visual is
+      /// attached to so that the rotation visuals are rotated to match the
+      ///  object's rotation. For world space rotation, set this to identity
```
Comment (ahcorde) [inline include/gz/rendering/GizmoVisual.hh:82]:
```suggestion
      /// object's rotation. For world space rotation, set this to identity
```

**Review by ahcorde — APPROVED** [review]
LGTM, just a minor comment

File: include/gz/rendering/GizmoVisual.hh (hunk 1 above)
Comment (iche033) [reply include/gz/rendering/GizmoVisual.hh]: [e2cc919](https://github.com/gazebosim/gz-rendering/pull/882/commits/e2cc91913cca6a32866f4f6fa6ecf7a2b76e4897)

**Comment by azeey** [comment]
Can we merge this after Harmonic?

**Comment by iche033** [comment]
> Can we merge this after Harmonic?

yes, no hurry

**Pull request merged**